JamJar/ConcertPageViewController.swift

```diff
@@ -0,0 +1,226 @@
+//
+//  ConcertPageViewController.swift
+//  JamJar
+//
+//  Created by Mark Koh on 3/12/16.
+//  Copyright © 2016 JamJar. All rights reserved.
+//
+
+import UIKit
+import SCLAlertView
+
+class ConcertPageViewController: BaseViewController, UITableViewDelegate, UITableViewDataSource, UISearchBarDelegate {
+    
+    @IBOutlet weak var tableView: UITableView!
+    @IBOutlet weak var artistImageView: UIImageView!
+    @IBOutlet weak var artistLabel: UILabel!
+    @IBOutlet weak var dateLabel: UILabel!
+    @IBOutlet weak var venueLabel: UILabel!
+    
+    var concert: Concert? = nil
+    var videos: [Video] = [Video]()
+    var myVideos: [Video] = [Video]()
+    var individualVideos: [Video] = [Video]()
+    var jamjars: [JamJarGraph] = [JamJarGraph]()
+    
+    var showJamJars: Bool = true
+    var showMyVideos: Bool = true
+    var showIndividualVideos: Bool = true
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Register the reusable video cell
+        self.tableView.registerNib(UINib(nibName: "VideoCell", bundle: nil), forCellReuseIdentifier: "VideoCell")
+        self.tableView.registerNib(UINib(nibName: "JamJarCell", bundle: nil), forCellReuseIdentifier: "JamJarCell")
+        self.tableView.registerNib(UINib(nibName: "JamJarHeaderCell", bundle: nil), forCellReuseIdentifier: "JamJarHeaderCell")
+        
+        // Set JamJars
+        ConcertService.getJamJars((self.concert?.id)!) {
+            (success, result, error) in
+            if !success {
+                // Error - show the user
+                let errorTitle = "JamJar error!"
+                if let error = error { SCLAlertView().showError(errorTitle, subTitle: error, closeButtonTitle: "Got it") }
+                else { SCLAlertView().showError(errorTitle, subTitle: "", closeButtonTitle: "Got it") }
+            }
+            else {
+                // Successfully retrieved JamJars, store them
+                self.jamjars = result!
+                self.tableView.reloadData()
+            }
+        }
+        
+        // Set myVideos
+        self.myVideos = self.videos.filter { (video) -> Bool in
+            return (video.user.username.lowercaseString == UserService.currentUser()?.username.lowercaseString)
+        }
+        
+        // Set individualVideos
+        self.individualVideos = self.videos.filter { (video) -> Bool in
+            return !(video.user.username.lowercaseString == UserService.currentUser()?.username.lowercaseString)
+        }
+        
+        if let concert = self.concert {
+            self.title = concert.getArtistsString()
```

Comment:
We probably wanna make this title more explicit than just the artist string, but I'm not sure how >.<  Maybe `Venue | ArtistString`?